TFNetwork.py

```diff
@@ -1,6 +1,8 @@
 
 from __future__ import print_function
 
+import pprint
```

Comment:
Do `from pprint import pprint` instead. Or better: Don't do that here at all. Do it locally in the method/function where you need it.